MovieApp/Scenes/Cell/CategoryCell.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  CategoryCell.swift
+//  MovieApp
+//
+//  Created by Ana Dzebniauri on 18.07.23.
+//
+
+import UIKit
+
+final class CategoryCell: UICollectionViewCell {
+    
+    //MARK: - Properties
```

Comment:
`MARK`-სა და `//`-ს შორის და ზოგადად `//`-სა და მის შემდეგ ტექსტს შორის გამოიყენე ერთი სფეისი ` `
```suggestion
    // MARK: - Properties
```